contentful/template-marketing-webapp-nextjs
# chore: [] graphql Error handling

**_What will change?_**

- Tweaked the GraphQL api client, it now does not retry failed calls, neither on focus nor on an interval
- On error, it now redirects pages to the 404 page.

To test this:
1. Create any page/blog/category in Contentful
2. Create a reference to a **new model** in one of the reference fields (probably SEO fields)
3. Publish both the page and the newly created model that is used as reference
4. Set the status of the newly created references model to `Draft`, this will trigger a broken query in the frontend.
5. Navigate to the newly created page
6. The result should be: An error warning in the browser console, and the page redirecting to the 404 page
